src/anonymizer/__mocks__/index.ts

```diff
@@ -25,5 +29,7 @@ export const proxify = (maybeUrl = ''): string => {
   return typeof maybeUrl === 'string' ? String(maybeUrl) : ''
 }
 
-// Config is not mocked
-export { config } from '../anonymizer.config'
+export const config = {
```

Comment:
config is now also mocked, otherwise unit test results depend on the current app configuration